wiki/Beatmap_information/en.md

```diff
@@ -0,0 +1,141 @@
+
+To the left side beneath the play count, the title together with the artist is shown. Clicking on the title will automatically search for other beatmaps with the exact same song title. Underneath the artist's name, an icon and the name of the mapper is displayed as well as the upload date and when the beatmap was last changed (e.g. when the beatmap was moved to a certain [category](/wiki/Beatmap/Category)).
+
+Further below these action buttons from left to right may be present, allowing the following:
+
+- **Favourite:** Add this beatmap to the favourites section in the user profile.
+- **Download:** Download the beatmap. If the beatmap has a background video, the user can choose to download the beatmap with or without the video.
+- **osu!direct**: Download the beatmap directly in the game client without requiring to manually open the file.
+- **Report:** If the beatmap is in either the [graveyard](/wiki/Beatmap/Category#graveyard), or the [WIP and the pending](/wiki/Beatmap/Category#wip-and-pending) category, the user has the ability to [report the beatmap](/wiki/Reporting_bad_behaviour#beatmap), should there be any rule-violating content. By clicking on the 3 dots, a dropdown menu appears, revealing the `report` button.
+
+### Statistics panel
+
+::: Infobox
+![](img/statistics-panel.png "Statistics panel showing the beatmap settings")
+:::
+
+To the right side of the difficulty menu is the statistics panel. Above the panel, the beatmap's category can be seen. Small icons will indicate if the beatmap has a video or a storyboard. When clicking the triangle-shaped button below the category, a short preview of the song will be played. Pause the preview by clicking a second time.
+
+Underneath the preview menu the song length, the BPM, as well as the number of [hit objects](/wiki/Gameplay/Hit_object) can be seen from left to right respectively. Upon hovering over the song length, a tooltip will additionally show the [drain](/wiki/Beatmap/Drain_time) length.
+
+Depending on the game mode, the following beatmap settings may be visible, along with the corresponding value next to it:
+
+| Setting | Description | Game mode |
+| :-: | :-- | :-: |
+| [Circle size](/wiki/Beatmap/Circle_size) (CS) | Determines how big the hit circles are. | ![][osu!] ![][osu!catch] |
+| [HP drain](/wiki/Beatmap/HP_drain_rate) (HP) | Defines how much health is gained and lost while playing. | ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania] |
+| [Approach rate](/wiki/Beatmap/Approach_rate) (AR) | Regulates the speed hit objects appear in. | ![][osu!] ![][osu!catch] |
+| [Accuracy](/wiki/Gameplay/Accuracy) | Measures how good the player hits hit objects on time. | ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania] |
+| Key count | Specifies the number of keys used when playing a beatmap. | ![][osu!mania] |
+
+At the bottom of the statistics panel is the [star rating](/wiki/Beatmap/Star_rating) detailing how much skill the player requires for the beatmap.
```

Comment:
Not *technically* correct considering SR evaluated maps' difficulty rather than players' skills. Best to refer to the map rather than the player.
```suggestion
At the bottom of the statistics panel is the [star rating](/wiki/Beatmap/Star_rating), which is an algorithmic abstraction of a map's difficulty.
```